lang/en.json

```diff
@@ -674,7 +674,9 @@
         "choose_an_option": "Please choose an option",
         "select_one": "Please select one",
         "description": "Description",
-        "retail_price": "MSRP",
+        "retail_price": "MSRP:",
+        "was": "Was:",
```

Comment:
Can we prefix value with `price` like `price_was`. Same for `now`.

Reply:
done